browser/decentralized_dns/test/BUILD.gn

```diff
@@ -36,8 +33,6 @@ source_set("unit_tests") {
     "//brave/browser/decentralized_dns/test/decentralized_dns_navigation_throttle_unittest.cc",
     "//brave/browser/decentralized_dns/test/utils_unittest.cc",
     "//brave/browser/net/decentralized_dns_network_delegate_helper_unittest.cc",
-    "//brave/net/dns/brave_resolve_context_unittest.cc",
-    "//brave/net/dns/dns_transaction_unittest.cc",
```

Comment:
we can drop `brave_dns_friend` now.

Reply:
done in https://github.com/brave/brave-core/pull/13354/commits/ac2744997363aabb5f8662f1dc05c053c3aa1a96